src/components/calcite-combobox/calcite-combobox.tsx

```diff
@@ -952,8 +951,8 @@ export class CalciteCombobox {
           aria-owns={guid}
           class={{
             wrapper: true,
-            "wrapper--active": open,
-            "wrapper--single": single
+            "wrapper--single": single || !this.selectedItems.length,
```

Comment:
Can you revert the other file changes if this is the only fix needed?

Reply:
there is one more line of code that's changed [:858](https://github.com/Esri/calcite-components/pull/3094/files#diff-3c5e42fd5fba0b231584840e4808faf46b8f2cdbda50c5b3bdd1b86dfa779378R858).